keanu-project/src/test/java/io/improbable/keanu/vertices/dbl/probabilistic/BetaVertexTest.java

```diff
@@ -38,22 +41,56 @@ public void setup() {
     }
 
     @Test
-    public void matchesKnownLogDensityOfScalar() {
+    public void logProbMatchesKnownLogDensityOfScalar() {
         BetaVertex tensorBetaVertex = new BetaVertex(2., 3.);
         BetaDistribution betaDistribution = new BetaDistribution(2.0, 3.0);
         double expectedDensity = betaDistribution.logDensity(0.5);
         ProbabilisticDoubleTensorContract.matchesKnownLogDensityOfScalar(tensorBetaVertex, 0.5, expectedDensity);
     }
 
     @Test
-    public void matchesKnownLogDensityOfVector() {
+    public void logProbGraphMatchesKnownLogDensityOfScalar() {
+        DoubleVertex alpha = ConstantVertex.of(2.);
+        DoubleVertex beta = ConstantVertex.of(3.);
+        BetaVertex tensorBetaVertex = new BetaVertex(alpha, beta);
+        LogProbGraph logProbGraph = tensorBetaVertex.logProbGraph();
+
+        LogProbGraphValueFeeder.feedValue(logProbGraph, alpha, alpha.getValue());
+        LogProbGraphValueFeeder.feedValue(logProbGraph, beta, beta.getValue());
+        LogProbGraphValueFeeder.feedValue(logProbGraph, tensorBetaVertex, DoubleTensor.scalar(0.5));
```

Comment:
can you pull the `0.5` out into a `value` variable so it's clearer it's being passed to both logprobs?

Reply:
I think it's easier to read actually with just the numbers, there are too many variables already

Reply:
ok